app/services/reachable_users_service.rb

```diff
@@ -110,4 +110,51 @@ def get_users_count
     query = ActiveRecord::Base.send(:sanitize_sql_array, [sql, params])
     ActiveRecord::Base.connection.execute(query).to_a.first
   end
+
+  def self.get_running_campaigns_for_user(user)
+    sql = <<~SQL.tr("\n", " ").squish
+      select
+      c.id as campaign_id,
+      c.available_doses,
+      COALESCE(cm.n_confirmed_matches, 0) as n_confirmed_matches,
+      COALESCE(um.n_user_matches, 0) as n_user_matches,
```

Comment:
Why return `n_user_matches` if it's always 0?

Reply:
Je l'utilisait juste pour debug

Reply:
👍